persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalFileConstants.java

```diff
@@ -35,11 +35,20 @@ private BalFileConstants() {
     public static final String CLOSE_RECORD_VARIABLE = "record {|%s|}";
     public static final String DOUBLE_QUOTE = "\"";
     public static final String SINGLE_QUOTE = "\'";
+    public static final String APOSTROPHE = "`";
     public static final String EMPTY_STRING = "";
     public static final String SPACE_STRING = " ";
     public static final String COMMA_SPACE_STRING = ", ";
+    public static final String FIELD_ACCESSS_STRING = ".";
+    public static final String ARRAY_TYPE_STRING = "[]";
+    public static final String START_CLOSED_RECORD_STRING = "{|";
+    public static final String END_CLOSED_RECORD_STRING = "|}";
+    public static final String START_RECORD_STRING = "{";
+    public static final String END_RECORD_STRING = "}";
+    public static final String COLON_STRING = ":";
+    public static final String SEMI_COLON_STRING = ";";
```

Comment:
Do we need "STRING" as a suffix?

Reply:
It's not required. Removed in [704a43f](https://github.com/ballerina-platform/persist-tools/pull/46/commits/704a43f3344de31ca790267f6ba9cc31ad09d26f)